src/calculateRentalCost.js

```diff
@@ -28,7 +28,23 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const PRICE_PER_DAY = 40;
```

Comment:
You can move the PRICE_PER_DAY constant outside of the function, as it's a constant that doesn't change between function calls.

Reply:
> You can move the PRICE_PER_DAY constant outside of the function, as it's a constant that doesn't change between function calls.

Thanks, I've already fixed it